components/Home/Bonding/BondingList/BondingList.jsx

```diff
@@ -67,6 +68,23 @@ const getColumns = (
       key: 'lpChainId',
       render: (x) => getNetworkName(x),
     },
+    {
+      title: 'Guide Link',
+      dataIndex: 'guide',
+      key: 'guide',
+      width: 100,
+      render: (x) => {
+        return (
+          <a
+            href={`${BOND_WEBSITE_URL}paths/${x || 'olas-eth-via-uniswap-on-ethereum'}`}
```

Comment:
shouldn't we just point to the `BOND_WEBSITE_URL` in case the guide is undefined?